cmd/main_test.go

```diff
@@ -40,6 +42,7 @@ type HTTPheader struct {
 
 var k8sClient client.Client
 var testEnv *envtest.Environment
+var ns1, ns2, ns3 k8sapi.Namespace
```

Comment:
I'm not sure what I think about this implementation in terms of scalability. would like another opinion